decidim-participatory_processes/app/controllers/decidim/participatory_processes/admin/moderations_controller.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ParticipatoryProcesses
```

Comment:
Yeah, I think for `moderations` and `categories` everything can stay in core and this controller is not necessary. I'll review it!